### DISCUSSION TITLE
Offline RSS support

### DISCUSSION THREAD

**Discussion by Twig6943** [spacecowboy/Feeder#341] [Ideas]
This would be pretty cool (not sure if this is a thing) (Auto downloading videos images text etc..)

**Comment by spacecowboy** [comment]
Text IS already downloaded.

Only images aren't downloaded ahead of time

**Comment by Twig6943** [comment]
is it impossible to download assets (audio,video,gifs etc)

**Reply by spacecowboy** [reply]
No, just isn't done today
